### PR TITLE
[Compression] support aten::sub & aten::constant_pad_nd in speedup

### PR DESCRIPTION
**Pull request by J-shang**
### Description ###
fix issue #4636
fix issue #4687
fix issue #4688

### CODE REVIEW

File: nni/compression/pytorch/speedup/jit_translate.py [hunk 1]
```diff
@@ -125,6 +125,27 @@ def add_python(node, speedup):
...
+    for i in range(2):
+        input_i = inputs[i]
+        debug_name = input_i.debugName()
+        if debug_name not in speedup.internal_result:
+            # this input is a constant value
+            # TODO: what if this input is a constant tensor
+
+            if input_i.toIValue() is not None:
+                constant = parse_constant(input_i, speedup)
+                break
```
Comment (QuanluZhang) [inline nni/compression/pytorch/speedup/jit_translate.py:141]: I am a little confused about this part. what is constant here?

Comment (J-shang) [reply nni/compression/pytorch/speedup/jit_translate.py]: constant here means maybe in your model, you have such operation:
```
output = torch.sub(x, 10)
```
here, `10` is constant.

File: nni/compression/pytorch/speedup/jit_translate.py [hunk 2]
```diff
@@ -125,6 +125,29 @@ def add_python(node, speedup):
...
+            if input_i.toIValue() is not None:
+                constant[i] = parse_constant(input_i, speedup)
+                break
+    if constant[0] is None and constant[1] is None:
+        new_sub = torch.sub
+    elif constant[0] is not None:
+        new_sub = partial(torch.sub, input=constant)
+    else:
+        new_sub = partial(torch.sub, other=constant)
+    return new_sub
```
Comment (J-shang) [inline nni/compression/pytorch/speedup/jit_translate.py:148]: I have no confidence about this part, pls help review @zheng-ningxin , is it possible that `input` and `other` are both constant?

Comment (zheng-ningxin) [reply nni/compression/pytorch/speedup/jit_translate.py:148]: I think it looks good.

File: nni/compression/pytorch/speedup/jit_translate.py [hunk 3]
```diff
@@ -362,7 +397,7 @@ def __init__(self, shape):
             logger.info('Reshape Module output size: %s', str(self.shape))
 
         def forward(self, *args):
-            return args[0].view(self.shape)
+            return args[0].reshape(self.shape)
```
Comment (zheng-ningxin) [inline nni/compression/pytorch/speedup/jit_translate.py:400]: Nice catch here!, I also found this bug.

**Review by zheng-ningxin — APPROVED** [review]

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**